azkaban-db/build.gradle

```diff
@@ -16,6 +16,19 @@
  */
 apply plugin: 'distribution'
 
+task testJar(type: Jar, dependsOn: testClasses) {
+    from sourceSets.test.output
+    classifier = 'test'
+}
+
+configurations {
```

Comment:
Based on this article, `testJar` can wrap the test.outputs to a jar format, then hide details.